examples/frontend/chat_session.py

```diff
@@ -59,9 +60,10 @@ def chat(self, question):
         server_message = []
         for chunk in chat_completion:
             chunk_message = chunk.choices[0].delta.content
-            if chunk_message is not None:
+            if chunk_message is not None and chunk_message != "":
                 if end is None:
-                    end = time.perf_counter()
+                    end = time.time()
+                    print(end)
```

Comment:
`print(end)` is a bit not user-friendly -- they have to manually do the difference to get how many time is spent on running the code.